ironic/drivers/modules/drac/raid.py

```diff
@@ -659,6 +743,12 @@ def _commit_to_controllers(node, controllers):
                   'job_id': job_id})
 
         driver_internal_info['raid_config_job_ids'].append(job_id)
+        driver_internal_info['raid_config_parameters'].append(raid_controller)
+        if raid_controller not in driver_internal_info[
+            'raid_config_parameters']:
+            driver_internal_info['raid_config_parameters'].append(
+                raid_controller)
```

Comment:
It looks like the if statement on line 747 above will always fail because the raid controller will already be in the list due to line 746.  As a result, it looks like lines 747-750 should be removed.

Reply:
Removed line 746. as discussed